cli/src/movement_util.rs

```diff
@@ -181,17 +180,7 @@ pub(crate) fn move_to_commit(
         .get_wc_commit_id()
         .ok_or_else(|| user_error("This command requires a working copy"))?;
 
-    let config_edit_flag =
-        match MovementSettings::try_from(command.settings().config())?.edit_mode() {
-            MovementEditMode::Always => true,
-            MovementEditMode::Never => false,
-            MovementEditMode::Auto => !&workspace_command
-                .repo()
-                .view()
-                .heads()
-                .contains(current_wc_id),
-        };
-
+    let config_edit_flag = command.settings().config().get_bool("ui.movement.edit")?;
```

Comment:
nit: you'll need to update `prev`/`next` help text.

Reply:
Whoops! Thanks for catching. Updated.